src/App.jsx

```diff
@@ -1,33 +1,43 @@
 import './App.scss';
-import { MoviesList } from './components/MoviesList';
+import { useState } from 'react';
 import moviesFromServer from './api/movies.json';
+import { MoviesList } from './components/MoviesList';
+
+export const App = () => {
+  const [query, onQuery] = useState('');
```

Comment:
```suggestion
  const [query, setQuery] = useState('');
```

we usually name this function starting with `set`
https://legacy.reactjs.org/docs/hooks-state.html